README.md

```diff
@@ -51,6 +51,109 @@ Example: check accounts in `account/samples/verify_ownership.yaml`.
 python -m account.verify_ownership --input account/samples/verify_ownership.yaml
 ```
 
+## block
+
+Running block extraction scripts requires the installaton of the local **block** package. This can be accomplished as follows:
```

Comment:
btw, it should be possible (and it is already - assuming you pip install all requirement files), to run the tools like
`PYTHONPATH=. python3 block/delegates/find_delegates.py`